plugins/modules/one_vm.py

```diff
@@ -196,6 +196,12 @@
       - Name of Datastore to use to create a new instace
     version_added: '0.2.0'
     type: str
+  updateconf:
+    description:
+      - When C(instance_ids) is provided, updates running VMs with the C(updateconf) API call.
```

Comment:
```suggestion
      - When I(instance_ids) is provided, updates running VMs with the C(updateconf) API call.
```
`instance_ids` is a module option, not a value.